data-es6.js

```diff
@@ -3570,8 +3591,9 @@ exports.tests = [
         var obj = { foo: 1, bar: 2 };
         var iterator = Reflect.enumerate(obj);
 
+        var passed = Symbol.iterator in item;
```

Comment:
I do not think the Safari is a modern browser :) About IE11 - I think, it will be added with `Symbol`.